test/e2e/proxy_publicviewer_test.go

```diff
@@ -255,7 +267,7 @@ func TestProxyPublicViewer(t *testing.T) {
 								cms := corev1.ConfigMapList{}
 
 								err = communityUserProxyClient.List(context.TODO(), &cms, client.InNamespace(sp.Status.ProvisionedNamespaces[0].Name))
-								require.True(t, meta.IsNoMatchError(err), "expected List ConfigMap as community user to return a NoMatch error, actual: %v", err)
+								require.ErrorContains(t, err, testCase.expectedError)
```

Comment:
Should we use `require.ErrorEquals()` instead?

Reply:
I've tried to explain the change from error equals to errorContains in the PR description. 
It's mostly because of the rest mapper change that the error is now returned wrapped from server.

Please let me know if something is unclear and can be improved.